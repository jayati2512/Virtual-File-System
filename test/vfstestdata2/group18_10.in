createvfs vfs10 10000
createvfs vfs11 10000
mountvfs vfs10
mountvfs vfs11
makedir /dir1/dir2/dir3/dir4 dir5
addfile /dir1/ file1.txt /usr/include/syscall.h
addfile /dir1/ file1.txt /usr/include/syslog.h
removefile /dir1/hello.txt
movefile /dir1/file1.txt /dir1/dir2/dir3
movedir /dir1/dir2/dir3/dir4 /dir1/dir2/dir3/file1.txt
exportfile /dir1/dir2/dir3/file1.txt /tmp/file2.txt
copyfile /dir1/dir2/dir3/file1.txt /dir1/file.txt
searchfile file /tmp/searchfile.txt
removefile /dir1/dir2/dir3/file1.txt
deletedir /dir1/dir2
listdir /dir5
listdir /dir1 1 /tmp/listdir.txt
unmountvfs vfs10
