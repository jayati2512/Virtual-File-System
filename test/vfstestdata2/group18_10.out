createvfs_SUCCESS
createvfs_SUCCESS
mountvfs_SUCCESS
mountvfs_FAILURE VFS_ALREADY_MOUNTED
makedir_SUCCESS
addfile_SUCCESS
addfile_FAILURE FILE_ALREADY_EXISTS
removefile_FAILURE CANNOT_FIND_SPECIFIED_FILE
movefile_SUCCESS
movedir_FAILURE DESTINATION_CANNOT_BE_FILE
exportfile_SUCCESS
copyfile_SUCCESS
searchfile_SUCCESS
removefile_SUCCESS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
listdir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
listdir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
unmountvfs_SUCCESS
